using_images/db_images/mongodb.adoc

```diff
@@ -134,3 +134,278 @@ The MongoDB image can be run with mounted volumes to enable persistent storage f
 
 * *_/var/lib/mongodb_* - This is the database directory where
 MongoDB stores database files.
+
+=== Using MongoDB replication
+
+IMPORTANT: The replication support provided by the MongoDB image is experimental and
+should not be used in production.
+
+The full source of replication example is available here:
+https://github.com/openshift/mongodb/tree/master/2.4/examples/replica
+
+IMPORTANT: The template provided in the example above does not use persistent
+storage. When you loose all members of the replication set, your data will be
+lost.
+
+This section describes how to start a cluster of MongoDB servers that implements
+master-slave replication and automated failover. This is the MongoDB’s
```

Comment:
s/the MongoDB's/MongoDB's/

or

"This is the recommended replication strategy for MongoDB."